### PR TITLE
Refactor jobseeker

### PR DESCRIPTION
**Pull request by Tsaikkari**
JobPostRow -> CustomRow

### CODE REVIEW

**Comment by Tsaikkari** [comment]
loading and error must be inside userInfo, otherwise you won't get to the jobPosts page on the CompanyProfile.

File: src/components/AddJobPost/index.tsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
 import React from 'react'
 
 import JobPostForm from '../JobPostForm'
+import Navbar from '../Navbar-logout'
```
Comment (Wyfy0107) [inline src/components/AddJobPost/index.tsx:4]: Could we add the `Navbar` in a more general place, like `user/profile` so it always stays visible ?

File: src/components/CompanyIcon/index.tsx [hunk 2]
```diff
@@ -4,9 +4,17 @@ type Icon = {
...
   return (
-    <img src={icon} alt="company-icon-svg" className="company-icon hide-md" />
+    <>
+      <img src={icon} alt="company-icon-svg" className="company-icon hide-md" />
+      {/* <Image 
+          src={`/${user.image}`} 
+          alt="company-image"
+          className="image" 
+          fluid /> */}
+    </>
```
Comment (Wyfy0107) [inline src/components/CompanyIcon/index.tsx]: Can we remove this?

File: src/components/JobseekerProfileForm/index.tsx [hunk 3]
```diff
@@ -1,101 +1,136 @@
...
+    workExperience: '',
+    image: '',
+    relocate: isOpenRelocate,
+    startingDate: '',
+  })
+
+  const KeyCodes = {
+    comma: 188,
+    enter: 13,
+  }
```
Comment (Wyfy0107) [inline src/components/JobseekerProfileForm/index.tsx]: This stays unchanged, so you can move it to outside the component

File: src/components/JobseekerProfileForm/index.tsx [hunk 4]
```diff
@@ -1,101 +1,136 @@
...
+      text: skill.name,
+    }
   })
 
-  // Handler for form inputs
-  const handleChange = (e: React.FormEvent<HTMLFormElement>) => {
-    const target = e.target
-    const value = (target as HTMLInputElement).value
-    const name = (target as HTMLInputElement).name
+  useEffect(() => {}, [suggestions, skills])
```
Comment (Wyfy0107) [inline src/components/JobseekerProfileForm/index.tsx]: This is empty

File: src/components/LogOut/index.tsx [hunk 5]
```diff
@@ -1,22 +1,31 @@
...
+  }
+
+  const styles = {
+    customButton: {
+      backgroundColor: '#6f42c1',
+      border: 'none',
+      color: '#fff',
+      borderRadius: '4px',
+    },
   }
```
Comment (Wyfy0107) [inline src/components/LogOut/index.tsx]: You can move this outside the component as well

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]

File: src/components/AddJobPost/index.tsx (hunk 1 above)
Comment (Tsaikkari) [reply src/components/AddJobPost/index.tsx:4]: We have navbar on the Tabs component but this is no tab, so we need navbar here.

File: src/components/CompanyIcon/index.tsx (hunk 2 above)
Comment (Tsaikkari) [reply src/components/CompanyIcon/index.tsx]: Done

File: src/components/JobseekerProfileForm/index.tsx (hunk 3 above)
Comment (Tsaikkari) [reply src/components/JobseekerProfileForm/index.tsx]: Ok.

File: src/components/JobseekerProfileForm/index.tsx (hunk 4 above)
Comment (Tsaikkari) [reply src/components/JobseekerProfileForm/index.tsx]: Seems to work fine without it now. Previously it didn't.

File: src/components/LogOut/index.tsx (hunk 5 above)
Comment (Tsaikkari) [reply src/components/LogOut/index.tsx]: Done.

**Review by Wyfy0107 — APPROVED** [review]
👍

**Pull request merged**